src/org/joml/Vector2d.java

```diff
@@ -676,32 +669,77 @@ public double lengthSquared() {
      * @return the length squared of the given vector
      *
      * @author F. Neurath
+     *
+     * @deprecated method name has a typo, use {@link #lengthSquared(double, double)} instead
```

Comment:
It's okay to fix/rename the method. I'd say this is the sort of breaking change is okay. Thanks for observing it!